style.css

```diff
@@ -36,6 +50,62 @@ body {
   background-color: #f7f7f9;
 }
 
+.nav-list {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .logo {
+    padding-top: 15px;
+  }
+
+  .nav-container {
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  .navibar {
+    display: flex;
+    width: 100%;
+    padding: 0;
+    background-color: #fff;
+  }
+
+  .menu-button {
+    display: none;
+  }
+
+  .nav-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 2px;
+    list-style-type: none;
+    padding: 0;
+  }
+
+  .navi-item {
+    text-align: center;
+  }
+
+  .navi-item a {
+    color: #344563;
+    font-family: "Poppins", sans-serif;
```

Comment:
- [Optional] Great job for the exact design👍👌I think your default font-family is `Poppins` so instead of using each place it is better to define it with  CSS universal selector `*` to minimize your code👍😁